updatecli/updatecli.d/plugin-compat-tester.yml

```diff
@@ -39,7 +39,8 @@ actions:
   default:
     kind: github/pullrequest
     scmid: default
-    title: Bump `plugin-compat-tester` version in prep.sh
+    title: Bump `plugin-compat-tester` version in prep.sh to {{ source `pctVersion` }}
     spec:
       labels:
+        - chore
```

Comment:
why not just use the standard dependencies? it's a project dependency

Reply:
An upgrade to PCT seemed like more of a build tooling change than a dependency update.  I'm happy to have either `chore` or `dependencies`.  I didn't want it being listed in the changelog as **"Other"**